src/azure-cli/azure/cli/command_modules/network/tests/latest/recordings/test_network_watcher_connection_monitor.yaml

```diff
@@ -7420,17 +8198,17 @@ interactions:
       ParameterSetName:
       - -n --source-resource -g --dest-resource --dest-port --tags
       User-Agent:
-      - python/3.7.4 (Windows-10-10.0.18362-SP0) msrest/0.6.10 msrest_azure/0.6.2
-        azure-mgmt-network/7.0.0 Azure-SDK-For-Python AZURECLI/2.0.76
+      - python/3.6.8 (Windows-10-10.0.18362-SP0) msrest/0.6.10 msrest_azure/0.6.2
+        azure-mgmt-network/9.0.0 Azure-SDK-For-Python AZURECLI/2.0.81
       accept-language:
       - en-US
     method: PUT
-    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/NetworkWatcherRG/providers/Microsoft.Network/networkWatchers/NetworkWatcher_westcentralus/connectionMonitors/cm1?api-version=2019-06-01
+    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/NetworkWatcherRG/providers/Microsoft.Network/networkWatchers/NetworkWatcher_westcentralus/connectionMonitors/cm1?api-version=2019-11-01
   response:
```

Comment:
FYI.

The API version is manually updated, and this is not a valid command's output.

Because some changes is made.
CMv1 is only allow to be created with api-version <= 2019-06-01, CMv2 is only allow to be created with api-version >209-06-01.

from now on, in this PR, only CMv1 is supported, CMv2 is at extensions, which will be brought in soon. So here manually update api-version make it right.